modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2505,8 +2505,14 @@ export class Wallet implements IWallet {
   private async prebuildTransactionTss(params: PrebuildTransactionOptions = {}): Promise<PrebuildTransactionResult> {
     const reqId = params.reqId || new RequestTracer();
     this.bitgo.setRequestTracer(reqId);
-    const apiVersion =
-      this._wallet.type === 'custodial' || this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? 'full' : 'lite';
+    if (
+      params.apiVersion === 'lite' &&
+      (this._wallet.type === 'custodial' || this.baseCoin.getMPCAlgorithm() === 'ecdsa')
+    ) {
+      throw new Error(`Custodial and ECDSA MPC algorithm must always use 'full' api version`);
+    }
+
+    const apiVersion = params.apiVersion || 'full';
```

Comment:
we should leave the default behavior the same, only overriding it with `params.apiVersion` if its explicitly provided.

e.g.
```
const apiVersion = params.apiVersion ||
 (this._wallet.type === 'custodial' || this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? 'full' : 'lite');
```